rizza/helpers/config.py

```diff
@@ -28,14 +28,14 @@ class Config():
     def __attrs_post_init__(self):
         """Load in config files, then environment variables"""
         # first, attempt to load nailgun config
-        self.base_dir = Path.home().joinpath('rizza')
+        self.base_dir = Path(os.getcwd())
         # we want to always use current directory as base for tests
         if 'tests' in str(self.cfg_file):
             self.cfg_file = Path().joinpath(self.cfg_file)
         elif self.cfg_file != str(Path(self.cfg_file).absolute()):
-            self.cfg_file = self.base_dir.joinpath(self.cfg_file)
-        self.RIZZA['CONFILE'] = self.cfg_file
+            self.cfg_file = str(self.base_dir.joinpath(self.cfg_file))
         self.load_config()
+        self.RIZZA['CONFILE'] = self.cfg_file
```

Comment:
I also played a bit with clean config `config rizza --clear`, to reload environment vars into the conf file and therefore to code.